source/index.html

```diff
@@ -1,12 +1,135 @@
+        </section>
+        <article>
+            <h2 class="visually-hidden">О системах питания для котов</h2>
+            <section class="index-main__sistems sistems">
+                <h3 class="visually-hidden">Виды питания</h3>
+                <ul class="sistems__list">
+                    <li class="sistems__item sistem">
+                        <div class="sistem__icon sistem__icon--slim">
+                        </div>
+                        <h4 class="sistem__title">похудение</h4>
+                        <p class="sistem__description">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на диету! Cat Energy Slim поможет вашему питомцу сбросить лишний вес.</p>
+                        <a class="sistem__link" href="/">каталог slim </a>
+                    </li>
+                    <li class="sistems__item sistem">
+                        <div class="sistem__icon sistem__icon--mass">
+                        </div>
+                        <h4 class="sistem__title">Набор массы</h4>
+                        <p class="sistem__description">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!</p>
+                        <a class="sistem__link" href="/">каталог pro</a>
+                    </li>
+                </ul>
+            </section>
+            <section class="index-main__how-it-work how-it-work">
+                <h3 class="visually-hidden">Как это работает</h3>
+                <ul class="how-it-work__list">
+                    <li class="how-it-work__item">
+                        Функциональное питание содержит только полезные питательные вещества.
+                    </li>
+                    <li class="how-it-work__item">
+                        Выпускается в виде порошка, который нужно лишь залить кипятком и готово. </li>
+                    <li class="how-it-work__item">
+                        Замените один-два приема обычной еды на наше функциональное питание.
+                    </li>
+                    <li class="how-it-work__item">
+                        Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+                    </li>
+                </ul>
+            </section>
+            <section class="index-main__example example">
+                <h3 class="example__title">Живой пример</h3>
+                <p class="example__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день. </p>
+                <dl class="example__results results">
+                    <dt class="results__characteristic">снижение веса</dt>
+                    <dd class="results_value">5 КГ</dd>
+                    <dt class="results__characteristic">затрачено времени</dt>
+                    <dd class="results_value">60 ДНЕЙ</dd>
```

Comment:
Ну здесь у вас банальные опечатки в названии класса. Правильно вот так:
```suggestion
                    <dd class="results__value">5 КГ</dd>
                    <dt class="results__characteristic">затрачено времени</dt>
                    <dd class="results__value">60 ДНЕЙ</dd>
```